src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -87,11 +87,11 @@
   </PropertyGroup>
 
   <!-- netcoreapp (not aot) uses the dotnet host. -->
-  <PropertyGroup Condition="'$(BuildingNETCoreAppVertical)' == 'true' AND '$(UseDotNetNativeToolchain)' != 'true'">
+  <PropertyGroup Condition="'$(UseDotNetNativeToolchain)' != 'true'">
```

Comment:
The original condition was added by @ViktorHofer in https://github.com/dotnet/buildtools/pull/2168/files#diff-425cb0dcf83ca9680e4a736c021dabe8R89 . Does everything in corefx still work without this condition?

Reply:
That's what I'm seeing in the CoreRT repo - it uses the MSIL "xunit.console.dll" app to run the AOT tests, it was just executing it using "call", not "dotnet"; this doesn't work well in VS2019 - according to JanK, we would need binding redirects as "call" launches the xunit.console.dll using the desktop .NET Framework, not .NET Core.